### PR TITLE
Fixes #4765 - new multiline continuation support

### LINKED ISSUE #4765: Update docs to explain how commands can be written to span multiple lines
PR PowerShell/PowerShell#8938 adds support for line continuance in pipelines with the pipe at the beginning of a line, with no backticks required. Documentation that describes how line continuance works needs to be updated to describe the additional methods that can be used to wrap a pipeline across multiple lines in a script.

The same documentation also needs to identify that when working interactively in the shell, line continuance with pipelines at the beginning of a line only works with copy paste when using ctrl+v to paste (right-click paste inserts lines one by one and with that method of paste, line continuance with pipes at the beginning of a line is not supported).

Additional supported methods to wrap pipelines with this PR in place:

```powershell
# Wrapping with a pipe at the beginning of a line (no backtick required)
Get-Process | Where CPU | Where Path
    | Get-Item | Where FullName -match "AppData"
    | Sort FullName -Unique
```

```powershell
# Wrapping with a pipe on a line by itself (this comes automatically with
# the addition of parser support for wrapping with a pipe at the beginning
# of a line since PowerShell already supported wrapping with a pipe at the
# end of a line)
Get-Process | Where CPU | Where Path
    |
    Get-Item | Where FullName -match "AppData"
    |
    Sort FullName -Unique
```

### PR DESCRIPTION
**Pull request by sdwheeler**
Fixes #4765 - new multiline continuation support
- about_pipelines
   - added note about pasting
   - cleaned up style, grammar, Acrolinx (86/96) for all versions
   - added v7 changes
- about_PSReadLine.md
   - added note about pasting to all versions

Fixes [AB#1607250](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1607250)

Version(s) of document impacted
------------------------------
- [x] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md [hunk 1]
```diff
@@ -49,129 +49,90 @@ Get-Process notepad | Stop-Process
 The first command uses the `Get-Process` cmdlet to get an object representing
 the Notepad process. It uses a pipeline operator (`|`) to send the process
```
Comment (davidsmatlak) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md]: Change to: **notepad.exe**

File: reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md [hunk 2]
```diff
@@ -180,86 +141,149 @@ Get-Process winlogon | Format-List -Property *
...
+
+### Parameter binding
+
+When you pipe objects from one command to another command, PowerShell attempts
+to associate the piped objects with a parameter of the receiving cmdlet.
+
+PowerShell's parameter binding component associates the input objects with
+cmdlet parameters according to the following criteria:
+
+- The parameter must accept input from a pipeline
```
Comment (davidsmatlak) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md]: Add periods to the list items.

Change was not to wasn't

File: reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md [hunk 3]
```diff
@@ -180,86 +141,149 @@ Get-Process winlogon | Format-List -Property *
...
 
-## One-At-A-Time Processing
+### One-at-a-time processing
 
 Piping objects to a command is much like using a parameter of the command to
-submit the objects.
-
-For example, piping objects representing the services on the computer to a
-`Format-Table` command, such as:
+submit the objects. Let look at a pipeline example. In this example, we use a
```
Comment (davidsmatlak) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md]: Change Let to Let's

File: reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md [hunk 4]
```diff
@@ -484,7 +420,7 @@ the `Get-Item` cmdlet and then the named values being bound to the
 ...
```
Comment (davidsmatlak) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md]: Add the Output tag.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md [hunk 5]
```diff
@@ -49,129 +49,90 @@ Get-Process notepad | Stop-Process
...
+      Format-Table -Property name, length
 ```
 
-This pipeline is comprised of four commands in the specified order. The
-command is written horizontally, but we will show the process vertically in
-the following graphic.
-
-`Get-ChildItem` `-Path` *.txt
+This pipeline consists of four commands in the specified order. The following
+illustration shows the output from each command as it is passed to the next
```
Comment (davidsmatlak) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md]: Use a contraction: it is > it's

File: reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md]: That doesn't work.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md (hunk 2 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md]: fixed

File: reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md (hunk 3 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md]: Fixed

File: reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md (hunk 4 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md]: fixed

**Pull request merged**